source/_integrations/netgear_lte.markdown

```diff
@@ -52,38 +52,38 @@ Messages arriving in the modem inbox are sent as events of type `netgear_lte_sms
 | `from`               | The sender of the message.
 | `message`            | The SMS message content.
 
-## Services
+## Actions
 
-### Service `netgear_lte.connect_lte`
+### Action `netgear_lte.connect_lte`
 
-This service asks the modem to establish its LTE connection, useful if the modem does not autoconnect.
+This action asks the modem to establish its LTE connection, useful if the modem does not autoconnect.
 
-| Service data attribute | Optional | Description |
+| Data attribute | Optional | Description |
 | ---------------------- | -------- | ----------- |
 | `host`                 | yes      | The modem that should connect (optional when just one modem is configured).
 
-### Service `netgear_lte.disconnect_lte`
+### Action `netgear_lte.disconnect_lte`
 
-This service asks the modem to close its LTE connection.
+This action asks the modem to close its LTE connection.
 
-| Service data attribute | Optional | Description |
+| Data attribute | Optional | Description |
 | ---------------------- | -------- | ----------- |
 | `host`                 | yes      | The modem that should disconnect (optional when just one modem is configured).
 
-### Service `netgear_lte.delete_sms`
+### Action `netgear_lte.delete_sms`
 
-The integration makes a service available to delete messages from the modem inbox. This can be used to clean up after incoming SMS events.
+The integration makes a action available to delete messages from the modem inbox. This can be used to clean up after incoming SMS events.
```

Comment:
```suggestion
The integration makes an action available to delete messages from the modem inbox. This can be used to clean up after incoming SMS events.
```